CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 - Add `id` field to Task
 - Add `queue` field to Task, defaulting to `default`
 - Remove `description` field from Task
+- Add client support for DELETE and PATCH requests
```

Comment:
I haven't updated the README with this, because `Task` doesn't have support for them yet. This PR was scoped to Schedules only